crates/uv-python/src/installation.rs

```diff
@@ -90,9 +93,16 @@ impl PythonInstallation {
         pypy_install_mirror: Option<&str>,
     ) -> Result<Self, Error> {
         let request = request.unwrap_or(&PythonRequest::Default);
+        let root_directory = crate::current_dir()?;
```

Comment:
I'm surprised this uses `current_dir()` still, `find_or_download` is used in a lot of places.